pkg/controller/nodenetworkconfigurationpolicy/conditions/policy_test.go

```diff
@@ -0,0 +1,181 @@
+package conditions
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+	enactmentconditions "github.com/nmstate/kubernetes-nmstate/pkg/controller/nodenetworkconfigurationpolicy/conditions/enactment"
+)
+
+type Enactment struct {
```

Comment:
Please make this private. The module is public (it is not `package conditions_test`), we don't want to expose these structures.

Also, functions in this package work with real types. Could you just use those here? Then you can test the labels an everything. But maybe I'm missing something and this way is better for testing?

Reply:
Those structs are not needed, I am going to just use nncp and nnce.